jest-setup.js

```diff
@@ -1,4 +1,5 @@
 import {NativeModules, AccessibilityInfo} from 'react-native';
+global.__UI_LIB_TESTING__ = true;
```

Comment:
Can we have less underscores 😬 

I think `global.UILIB_TESTING` or `global._UILIB_TESTING` is enough